api/transport/handler_test.go

```diff
@@ -54,21 +53,21 @@ func TestHandlerSpecLogMarshaling(t *testing.T) {
 	}{
 		{
 			desc: "unary",
-			spec: NewUnaryHandlerSpec(unaryHandlerFunc(func(_ context.Context, _ *Request, _ ResponseWriter) error {
+			spec: NewUnaryHandlerSpec(UnaryHandlerFunc(func(_ context.Context, _ *Request, _ ResponseWriter) error {
```

Comment:
I know these aren't your changes, but you could drop the underscores since none of the parameters are being used.

Fine to leave as is, since it's sort of unrelated